testsuite/forge-cli/src/main.rs

```diff
@@ -1718,13 +1718,16 @@ impl Test for RunForever {
     }
 }
 
-#[async_trait::async_trait]
-impl AptosTest for RunForever {
-    async fn run<'t>(&self, _ctx: &mut AptosContext<'t>) -> Result<()> {
+impl NetworkTest for RunForever {
+    fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
         println!("The network has been deployed. Hit Ctrl+C to kill this, otherwise it will run forever.");
         let keep_running = Arc::new(AtomicBool::new(true));
         while keep_running.load(Ordering::Acquire) {
-            thread::park();
+            if let Err(x) = ctx.runtime.block_on(ctx.swarm.health_check()) {
```

Comment:
ok, that makes sense, should be easy to rework it into a new mode and leave the old mode as is

Reply:
I was having trouble where `aptos-node` was crashing on other changes I made, but failing silently because forge just kept running and didn't print any warning/error